addons/website/static/src/js/tours/rte.js

```diff
@@ -18,11 +18,14 @@ tour.register('rte_translator', {
     content: "load french",
     trigger: '.modal-footer button:first',
     extra_trigger: '.modal select[name="lang"]:propValueContains(fr_BE)',
+}, {
+    content : "click language dropdown",
+    trigger : '.js_language_selector .dropdown-toggle',
+    timeout: 60000,
```

Comment:
I doubt opening the dropdown can take one minute. I think the timeout should stay at the next step ?

Reply:
Right I'll check 👍

Reply:
Probably 1 minute is the time load language and not to open a dropdown. So I think we have to keep this change.

Reply:
loading the language is on the next step

Reply:
Nope it's selection of the language from website. I'm talking about waiting for language to be loaded from backend modal : https://github.com/odoo/odoo/pull/22298/files#diff-d0d18ab9956e52fa3773990e45151243R20 . We wait for the language to be loaded, than redirect to website, and then select the language from dropdown.

Reply:
Oh, yeah. I was thinking the timeout was how long the step could take but it is how long we might have to wait before executing the step. My bad.